Sources/RedisMulti+Set.swift

```diff
@@ -0,0 +1,590 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+
+/// Extend Redis by adding the Set operations
+extension RedisMulti {
+    
+    /// Add one or more members to a set
```

Comment:
Yes. The Redis docs don't document the individual commands in the scope of a transaction. Please add the sentence I asked for, or do what you in the transactional sorted sets, which I kind of prefer as it matches the style of the other transactional stuff.